src/main/java/net/sf/mzmine/modules/masslistmethods/ADAPchromatogrambuilder/ADAPChromatogram.java

```diff
@@ -69,6 +66,9 @@
   // Top intensity scan, fragment scan
   private int representativeScan = -1, fragmentScan = -1;
 
+  // All MS2 fragment scan numbers
+  private int[] allMS2FragmentScanNumbers = new int[] {-1};

```

Comment:
I think it makes more sense to use an empty array here (remove the -1).

Reply:
okay